backend/pkg/instances/handler.go

```diff
@@ -171,9 +240,85 @@ func DeleteContainerHandler(c *gin.Context) {
 	})
 
 	if err != nil {
-		util.ThrowServiceUnavailableException(c, "Could not schedule container at the current time")
+		util.ThrowServiceUnavailableException(c, "Could  schedule container deletion at the current time")
 		return
 	}
 
+	err = db_instances.DeleteInstanceById(userId, containerId)
```

Comment:
Decide which naming scheme the function parameter should have. UserID and UserSub is not very intuitive.